### PR TITLE
Merge updated addindividual into master. 

### PR DESCRIPTION
**Pull request by ceeb950**
Changed indludeimmune to includeimmune.

### CODE REVIEW

**Comment by ceeb950** [comment]
@richfitz @giovannic updated a few bits and pieces

File: R/sirmodel.R [hunk 1]
```diff
@@ -271,7 +271,7 @@ individual_S_to_I <- function(S, I, human, immunity, age, location, pars = NULL)
         infected <- susceptible[sample.int(length(susceptible), n_to_infect)]
         api$queue_state_update(human, I, infected)
     }
-    if(pars$indludeimmune){
+    if(pars$includeimmune){
```
Comment (richfitz) [inline R/sirmodel.R]: Can you please add spaces around the if statement here and elsewhere like

```
if (pars$includeimmune) {
```

File: tests/testthat/test-sirmodel.R [hunk 2]
```diff
@@ -99,8 +99,8 @@ test_that("test that no one is infected if I is 0 at t = 0", {
 })
 
 test_that("test that an empty simulation exits ok for individual model", {
+
   library(individual)
```
Comment (richfitz) [inline tests/testthat/test-sirmodel.R]: Remove all library calls from test please

**Review by richfitz — CHANGES_REQUESTED** [review]
Minor style changes

**Comment by ceeb950** [comment]
@richfitz all checks passed on new commit - over to you!

**Review by richfitz — APPROVED** [review]

**Pull request merged**